http/dotnet/httpclient/src/Microsoft.Kiota.Http.HttpClientLibrary.csproj

```diff
@@ -2,18 +2,19 @@
   <Sdk Name="Microsoft.DotNet.PackageValidation" Version="1.0.0-preview.7.21379.12" />
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
This seems to break compilation of this project as net standard does not support the following we currently use
- `HttpContent.ReadAsByteArrayAsync()` does not have an overload with a CancellationToken parameter as we currently use it.
- `HttpRequestMessage` does not have the `Options` property but uses the `Properties` property
- We might manaully need to import some packages included by default in net5.0